scripts/testing/city_runner/test_info.py

```diff
@@ -156,12 +174,17 @@ def from_file(cls, list_file_paths, disabled_file_path, ignored_file_path, prefi
 
                     if len(chunks) >= 3:
                         attribute = chunks[2]
+                        print("___CSD__ attribute=", attribute)
                         if attribute == 'Ignore':
                             ignored = True
                         elif attribute == 'Disabled':
                             disabled = True
                         elif attribute == 'Unimplemented':
                             unimplemented = True
+                        elif attribute == 'Xfail':
+                            xfail = True
+                        elif attribute == 'Mayfail':
+                            mayfail = True
```

Comment:
While we're making changes here, do you think we could make these attributes case-insensitive? For consistency with the existing attributes, absolutely we should accept `Xfail` but for ease of typing I'd like it if we could accept `xfail` too. But I'm okay leaving it like this too if there's any reason to not do that.